src/Dependencies/xUnit.net/xUnit.net.csproj

```diff
@@ -44,13 +44,17 @@
       <CopyLocalAssembly Include="xunit.runner.utility.dotnet">
         <RelativePath>xunit.runner.utility\$(XUnitVersion)\lib\dotnet\xunit.runner.utility.dotnet.dll</RelativePath>
       </CopyLocalAssembly>
+      <CopyLocalAssembly Include="fluentassertions">
```

Comment:
Yes, I will fix it soon, but I'm splitting in to steps as it's easier to work through the issues.